app/services/food_service.rb

```diff
@@ -0,0 +1,17 @@
+class FoodService
+  class << self
+    def call_for_a_food(food)
+      response = conn.get("/fdc/v1/foods/search?&query=#{food}&api_key=#{ENV['food_api_key']}")
```

Comment:
Would be better to set the params in a block here. Something like:
```
conn.get("/fdc/v1/foods/search") do |req|
   req.query = food
   req.api_key = ENV['food_api_key']
end
```

Much more readable and maintainable like this.